scripts/aws-device-farm/s3-client.mjs

```diff
@@ -0,0 +1,7 @@
+import { S3Client } from "@aws-sdk/client-s3";
+
+export function getS3Client() {
+  return new S3Client({
+    region: "eu-central-1",
```

Comment:
Just wanted to call out that the device farm region is `us-west-2`. Should we have them on the same region, or do we want to test with them in different regions for more coverage?

Reply:
Our S3 bucket is in eu-central-1.

Reply:
Yeah the AWS Device Farm client uses `us-west-2`. 👍